posthog/api/team.py

```diff
@@ -219,6 +219,7 @@ class Meta:
             "capture_dead_clicks",
             "user_access_level",
             "default_data_theme",
+            "revenue_tracking_config",
```

Comment:
I'm surprised we didn't need to update those `.ambr` files 🤔

Reply:
Maybe because the migration was already merged?

Reply:
Ah, lol, probably :)